ykman/cli/fido.py

```diff
@@ -176,23 +219,54 @@ def try_reset(controller_type):
             controller = ctx.obj['controller']
             controller.reset(touch_callback=prompt_for_touch)
 
-    try:
-        try_reset(Fido2Controller)
-    except CtapError as e:
-        if e.code == CtapError.ERR.ACTION_TIMEOUT:
-            ctx.fail(
-                'Reset failed. You need to touch your'
-                ' YubiKey to confirm the reset.')
-        elif e.code == CtapError.ERR.NOT_ALLOWED:
-            ctx.fail(
-                'Reset failed. Reset must be triggered within 5 seconds'
-                ' after the YubiKey is inserted.')
-        else:
+    if ctx.obj['dev'].is_fips:
+        if not force:
+            click.echo(
+                '\nWARNING: This will destroy the FIPS attestation key, making '
+                'this YubiKey permanently incapable of FIPS mode.'
+            )
+            destroy_input = click.prompt(
```

Comment:
I don't like this, we don't do it anywhere else and there is a big warning anyway. I suggest a regular prompt. If the user is doing `fido reset` from the command line on a FIPS key (not a consumer device), they hopefully know what they are doing.

Reply:
Good points... maybe we should ask TPM about this. I wouldn't be so sure that it's clear that resetting is irreversible (in the sense of resetting the attestation key).